docs/design/DeepArchiveAPIsSupport.md

```
@@ -0,0 +1,357 @@
+# Support NooBaa Glacier APIs to connect to DeepArchive-S3
+
+
+### Table of Contents
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)